spec/support/custom_rspec_matchers.rb

```diff
@@ -0,0 +1,9 @@
+module CustomRSpecMatchers
+  def have_table_heading(text)
+    have_selector(".govuk-table__header", :text => text)
+  end
+
+  def have_step_text_in_row(row:, text:)
+    have_selector(".govuk-table__body .govuk-table__row:nth-child(#{row}) .step-list__title", :text => text)
```

Comment:
This is check is specific to step-by-steps, and as it's only need for the steps list, I think this methods would be better placed in spec/features/managing_step_by_step_pages_spec.rb